### PR TITLE
Quick fix for the yaml file parsing

### PR DESCRIPTION
**Pull request by cici37**
#### What type of PR is this:
/kind bug

#### What this PR does / why we need it:
This is a bug introduced from https://github.com/kubernetes/sig-release/pull/1584, and causing issue when generating release note for v1.22-alpha3: https://github.com/kubernetes/release/issues/2114

#### Which issue(s) this PR fixes:
https://github.com/kubernetes/release/issues/2114

### CODE REVIEW

File: releases/release-1.22/release-notes/maps/pr-101496-map.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 pr: 101496
 releasenote:
-  text: `podAffinity`, `NamespaceSelector` and the associated `CrossNamespaceAffinity`
-    quota scope features graduate to Beta and they are now enabled by default.
+  text: "`podAffinity`, `NamespaceSelector` and the associated `CrossNamespaceAffinity`
```
Comment (pmmalinov01) [inline releases/release-1.22/release-notes/maps/pr-101496-map.yaml:3]: Can you put a word before  `podAffinity`, something like The or A ?

Comment (cici37) [reply releases/release-1.22/release-notes/maps/pr-101496-map.yaml]: Updated. Would you mind to review again? Thank you

**Pull request merged**